lms/templates/problem.html

```diff
@@ -13,10 +13,15 @@ <h2 class="problem-header">
     </div>
   <div class="action">
     <input type="hidden" name="problem_id" value="${ problem['name'] }" />
-
+    % if show_demand_hint:
+    <div class="problem-hint" aria-live="polite">${demand_hint}</div>
```

Comment:
@nparlante I was just testing this feature with screen readers and I noticed that the hint is not being announced, despite the aria-live here.  Is this ```<div>``` here during the original render, or only inserted after the Hint button is pressed?  ```aria-live``` regions cannot be dynamically inserted.  The screen reader watches these divs for changes.  HOwever, if the change occurs at the same time the div is added to the DOM, the screen reader doesn't know its there.   Can you make this div render at the original page load, and then inject the hint into it?  Also, it would be nice to add a unique ID to this div and then add an ```aria-describedby``` attribute to the form input element that references the unique ID of the hint.  This would read the hint aloud when the user moved focus back to the form input as a description.  Consider the following:
I'm a screen reader user filling out a text input problem "What color is the sky?"  I enter "red" and then move focus to the Hint button.  A hint appears that says "its the same color as a bluejay".  I realize my mistake and move my focus back to the text input.  The screen reader will read "edit text, what color is the sky, its the same color as a bluejay, red"
(type, label, description, current value)